cwms-data-api/src/test/java/cwms/cda/api/LevelsControllerTestIT.java

```diff
@@ -808,4 +1266,63 @@ enum GetAllTestNewAliases
             _expectedContentType = expectedContentType;
         }
     }
+
+    private String createRatingSpec(String locationName) throws Exception
```

Comment:
Preferably these would've gone in the DataApiTest class, but this at least makes it easier for the next person to yank up and make more generic.